### PR TITLE
Save to pptx

### PR DESCRIPTION
**Pull request by seb231**
Trying to make sure these are as small and concise as possible. Also trying to ensure I bring in the bare minimum dependencies.

### CODE REVIEW

File: src/kixi/plugsocket.clj [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "Save the workbook into a stream.
+  The caller is required to close the stream after saving is completed."
+  [^OutputStream stream ^XMLSlideShow powerpoint]
+  (assert-type powerpoint XMLSlideShow)
+  (.write powerpoint stream))
+
+(defn save-powerpoint-into-file!
+  "Save the workbook into a file."
+  [^String filename ^XMLSlideShow powerpoint]
+  (assert-type powerpoint XMLSlideShow)
```
Comment (otfrom) [inline src/kixi/plugsocket.clj:22]: is `assert-type` needed for these given that the multimethod is switching on the class?

**Review by otfrom — APPROVED** [review]
approved, not sure the assert-type is needed though given the multimethod

File: src/kixi/plugsocket.clj (hunk 1 above)
Comment (seb231) [reply src/kixi/plugsocket.clj:22]: That was torn from kixi.large.legacy, so you'll have to tell me?

Comment (otfrom) [reply src/kixi/plugsocket.clj:22]: kixi.large.legacy is just copying over the stuff from docjure w/o changing anything that wasn't needed to update the underlying poi libs.

Comment (seb231) [reply src/kixi/plugsocket.clj:22]: Will remove in following PR as not required

**Pull request merged**